src/bundle/Resources/public/scss/_main-menu.scss

```diff
@@ -9,22 +9,17 @@
     }
 
     display: flex;
-    height: 100%;
-    min-height: calc(100vh - #{calculateRem(72px)});
-    background-color: $ibexa-color-black;
+    height: calc(100vh - #{calculateRem(72px)});
```

Comment:
I just wonder: do we need this?

Reply:
in my opinion yes because it is 100vh - the height of the header